src/main/resources/plugin-descriptor.properties

```diff
@@ -0,0 +1,34 @@
+### mandatory elements for all plugins:
```

Comment:
This is the definition of the plugin in 2.0.0. I opted to keep the comments from the example file for context, but could also be removed I guess